launch_ros/launch_ros/py.typed

```diff
@@ -0,0 +1 @@
+
```

Comment:
The `py.typed` file in `rclpy` is completely empty, I would just prefer if we were consistent here.  So I'd say let's remove the carriage-return that is in here.

Reply:
done